server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -156,22 +156,13 @@ public String getQueueName() {
   private final AtomicBoolean compactionRunning = new AtomicBoolean(false);
 
   protected Compactor(CompactorServerOpts opts, String[] args) {
-    super("compactor", opts, args);
-    queueName = opts.getQueueName();
-    aconf = getConfiguration();
-    setupSecurity();
-    watcher = new CompactionWatcher(aconf);
-    var schedExecutor =
-        ThreadPools.getServerThreadPools().createGeneralScheduledExecutorService(aconf);
-    startGCLogger(schedExecutor);
-    startCancelChecker(schedExecutor, TIME_BETWEEN_CANCEL_CHECKS);
-    printStartupMsg();
+    this(opts, args, null);
   }
 
   protected Compactor(CompactorServerOpts opts, String[] args, AccumuloConfiguration conf) {
     super("compactor", opts, args);
     queueName = opts.getQueueName();
-    aconf = conf;
+    aconf = conf == null ? super.getConfiguration() : conf;
```

Comment:
same comment about `aconf` variable applies here

Reply:
For completeness, same reply :smiley_cat: